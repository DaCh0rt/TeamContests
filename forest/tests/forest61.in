761 871
-7622 -5923
5804 -6750
-2295 1342
2290 4338
7484 -1630
-7791 -5251
7035 -3695
773 7695
-8293 -1481
-7953 -6200
2172 9713
5351 8222
1488 6503
8514 -4102
-5816 9941
4139 -4498
-7056 -3451
9302 9308
-7353 8311
-63 -5269
-1334 -970
-6673 7046
-3170 -826
-5792 -8160
-5745 -4854
-966 9390
9327 8874
-1757 4122
-4309 6101
-8397 2741
-734 3270
1787 2687
3505 -2400
-4326 6317
-3366 -405
-5850 5055
-4315 -5212
6755 -6407
-6136 -8928
-4765 5661
5293 -4115
4731 -7162
83 -715
5423 -2208
6423 -9165
2149 9722
5068 -6914
-8692 -8456
9739 3060
-8463 -7235
7962 2565
-4703 1066
6513 2167
-9770 -8636
-5455 -1620
1297 2409
4748 -5337
-4090 1690
-3716 1685
299 5953
-2028 3654
479 5046
5245 -5798
-454 8035
8804 -6989
1675 -2095
-9079 1146
2298 8133
-9000 -2660
-1473 25
7952 -950
3703 -3967
-6602 3405
-9695 6621
-9457 -9492
-8279 -9871
-9804 -7240
-2823 -8207
-9395 5317
-2523 1712
-1161 -5268
-6996 8460
7812 7875
-1641 4153
8309 -1746
2135 1895
-951 -2885
5124 -8431
4222 -1936
-7995 2373
5356 3696
-4339 1545
1040 7732
5701 -1300
2410 9763
-1071 4101
4701 8703
-7975 9683
-8817 -2260
-1807 -7523
6181 9915
-2511 649
-5025 -7476
1187 -5821
9109 -693
1397 -191
-6874 -718
8457 2547
-3402 3362
-1451 -4571
-5094 5199
5621 -6775
-3722 8959
-1693 -1994
4677 8621
-4988 2536
-2508 9855
9347 4870
-2365 -966
-3165 -9436
-7988 7876
-9879 9058
8725 -249
9901 -5371
6775 1504
9780 -6215
1997 -1026
-9607 1079
1401 2192
-7940 4707
5798 7619
8261 8753
-2933 9398
-1282 892
-5417 -1411
-8990 9321
8608 -9914
-4646 -8698
9706 2682
1532 -1554
-8856 -4094
-3216 -4709
9409 -8261
-8681 -7765
-8713 -3597
1841 -856
7251 -6354
-5498 4424
6384 3669
-742 7853
6222 6804
5008 8849
4498 8812
-3501 -134
7021 -8589
3494 2022
-8928 7597
-2518 8436
-4779 -3181
4933 7084
-5795 -5986
-1543 7175
2405 2815
5057 6575
-3453 3831
-7505 -7734
-8293 -3366
5589 4060
-2996 1224
6552 2610
-9999 2875
7813 7319
6503 -9407
-1229 5562
-3789 -8444
-1088 5340
-1629 -986
-1416 -1079
333 -7629
-9405 1675
3840 -6794
-311 8331
7204 -7456
7666 -7839
9511 -8891
-7851 1439
-1393 7647
-9735 -2586
4561 9758
-9104 -1725
9974 -2245
4485 -733
-9374 7267
9136 7159
7781 705
-6557 -870
-4822 8316
-8275 -4054
-1649 -8091
8519 9222
1745 -3739
9635 -3282
3295 -1096
-4899 2624
3436 -1923
5108 8580
6110 7503
9950 6180
6600 476
3472 5699
6796 2220
-4689 6680
9828 -2177
-2062 6693
-5055 6319
-4637 9697
8169 -8004
-4324 7778
877 -5431
-8659 9122
4480 3667
-4713 -2755
-1853 7375
1712 7661
-3283 -499
-1577 -2671
9341 334
-875 -4315
-1888 -931
-209 6907
5459 -6616
9909 -2201
-473 -4476
7266 -9849
7728 -6779
1100 730
-6455 5042
3907 -5449
6892 3419
-4634 -8721
-3511 4922
-8385 4727
-8290 -8279
-6557 221
330 -1656
-8053 -7967
6771 3018
3298 1981
-5595 6368
5196 6961
9138 -2319
9369 9072
-5825 1857
-2980 -6672
-6964 -2383
-19 9970
-7114 3487
-521 4205
5266 -3248
-3698 -8413
6350 -5561
8032 9486
4980 -9570
-4988 -7897
8342 4458
-3799 -6929
8609 4049
2019 -8739
-2352 9705
2760 8017
6629 -5060
1399 2320
3913 4843
-7018 -4640
-3843 2202
-6712 -2862
-5174 500
-4553 3207
5359 6557
-4517 9535
4660 -6829
-9336 -6041
-5399 -2497
5225 -9593
9184 -2365
-9116 9528
-6346 -4742
7194 7843
-1135 5015
7725 -5881
-8637 2204
8147 7886
3271 -8419
-8857 -3110
-8217 -2100
-1287 -2111
8379 34
1372 967
-8278 -7023
7820 9985
-7152 -2481
-7480 -8046
9053 -2297
-8731 3082
5231 -2318
-3287 -1800
-5618 -9759
-1301 9336
-6693 -2537
-4173 9247
-6772 2553
-8415 2327
-4924 -3996
-8533 3867
8348 -9638
-7899 5889
7897 -8091
9681 9030
-2797 2149
5743 -8194
-6806 8037
-5040 3357
-9152 4003
-7779 -9390
8482 -6760
4824 7561
-6172 3971
-1050 -7221
3619 6142
8938 4565
-8429 -1627
-9035 -2088
7401 461
4168 -7256
5420 407
3020 953
-315 1593
-5099 7349
-1949 -711
-2554 -6940
-4396 2848
-6894 1802
-9244 6605
3205 2363
-5009 2033
3850 8194
8446 -931
3105 6217
-5646 -9269
-7966 -1114
3501 -4511
-2143 9103
-192 -2424
-6518 4576
3724 8108
-1841 -2845
-1939 7886
8945 -2195
-7482 1582
-2080 -6105
6447 9114
756 -1912
2458 5877
-8693 -33
311 8706
4033 -7693
-716 -53
6544 8578
-1634 -1021
4925 1482
-4245 -6548
4487 -805
-4443 -1827
-8243 -5830
8301 -4131
-455 2125
2226 9846
4454 878
6380 2859
2368 -8556
594 6377
6257 -694
4111 4502
-6271 -3977
-3763 4099
6455 -7525
-3861 4294
-9485 5699
-9033 -5518
3464 -8111
-4835 -3142
5188 6417
-6428 -3108
-738 3404
7480 -498
6130 -6715
-4041 69
-6586 8605
3263 2760
-4290 7279
-4692 -5884
-7427 60
5997 -5580
-335 3574
1598 -5297
-5719 -6390
-4154 -6199
2207 7187
-6019 -821
-336 8392
-8043 6589
-6982 4714
-2728 -5873
-9747 154
7584 3446
9551 7763
-2644 -2813
9895 3432
-7777 -4962
-323 7857
-1142 9038
5733 7525
998 3849
4674 4819
-1803 -8610
-4803 6012
-8386 6136
-5371 6046
2384 -2949
8219 -298
-5829 -7476
-189 4906
3806 3162
-3328 -2924
1981 3032
7131 5262
7907 -9239
1275 2407
7980 -3907
1332 6237
2037 -4697
-2984 -4181
-3800 -882
2205 -2655
4705 550
-7674 -5411
3973 -3320
5071 -3274
-4485 -6468
-7582 -4660
2369 2069
1340 3929
1073 -1327
4337 5702
8734 -2726
-1917 5566
-2936 5775
-7844 3021
1833 -3246
-5148 6877
7825 6920
-2230 4491
-6977 9753
-7888 6922
-6932 4148
-6068 -688
2819 -5641
-5151 -7482
-4216 -7461
1244 7939
-3102 315
7685 -4658
1133 -9839
-4501 -8084
6351 -4231
657 3945
923 -1130
6802 -9621
1651 -5918
1260 -9937
-9167 -3933
5047 5540
-9033 1526
-1693 3638
-1266 -8999
8140 -1495
-6721 9907
7659 7674
-2154 -6760
7154 9281
-4096 9328
-3643 -9947
4543 -594
-2396 9298
-666 -7104
1843 -8911
5005 3222
9697 5152
964 4190
4032 6155
-6311 -5993
-287 4015
1727 -7958
-9313 -972
-2762 -6454
-4852 -2176
-3150 8644
1453 -6319
4044 -3871
-6329 -7243
-2221 3496
4422 1644
-5172 -644
-7374 -5860
727 -5172
4146 589
-1462 217
623 2350
-866 2286
7614 611
2214 -5811
9460 3996
-8141 -5419
7014 -2325
2702 -3177
3776 4030
9491 1038
-1151 699
-9843 -246
1097 -9149
9371 2493
-5725 4785
9028 38
-7800 2480
3251 -4811
-6949 -803
-6876 -788
-1164 7352
-5505 -8775
1625 -1799
-730 -811
-7139 2745
-8656 8150
-7202 -392
1181 -6382
-6516 3485
4152 2721
-3024 -1943
-9535 1147
-7032 9551
4354 -7889
7850 -8891
2790 -4609
-449 -2130
-2752 6320
9579 -5035
-8838 -8502
9146 -8297
-3966 -8516
-4237 9653
4157 -6804
9685 1915
-4377 6155
6174 5832
9509 6279
-1105 406
-5930 -911
-7219 1120
6708 -8446
4970 6894
1335 8997
-8403 5981
8481 -6787
-2268 -4723
9905 -7176
7742 1790
28 2618
-4336 -7230
4507 5497
-6605 -4040
-3428 3497
-5493 -9254
-4772 417
6471 -3644
5261 3771
-8773 -929
4454 -8137
-3496 -4651
-5790 -8219
-9024 -8733
-3313 -4984
-8712 7291
870 6930
5400 -3886
-1921 9375
-7874 -7243
-8933 9606
-3248 -8188
1038 -6892
-9529 655
-8254 778
-5133 -2239
-6135 -5234
297 -4696
1539 4878
-9142 6868
-562 2472
1226 -5501
-7585 3823
-7812 929
1200 748
-8591 3497
7575 3726
-1437 3909
2381 -5285
-1716 -2671
7775 -3008
-5297 6875
4074 -9904
-9712 -9247
9172 8048
-1031 9090
1355 -1287
-8680 -2938
-4837 8606
2555 3503
9513 8786
2190 4649
-6815 5630
631 3743
1650 4564
7774 -7465
-4667 -9630
-8381 -8693
-5966 -4347
-3760 5982
-473 1083
-4829 5318
-8460 -2887
5308 3531
3693 -53
8313 -6254
8924 1298
8861 -4655
5481 9850
7218 9596
9665 -9560
895 -752
-9090 5709
-4578 -3246
-2729 2736
-5485 -8529
4155 -199
4234 1119
6326 -9370
-3568 -481
987 -6634
9778 -5560
3144 -1261
-3092 8646
-8097 8628
7021 3012
3189 6383
3237 4914
-7865 -2099
3371 9797
-2766 3145
5276 1525
-5779 -5262
-696 -278
-4600 9647
-2132 6362
-2951 -8477
-3587 4217
-1767 -3120
-193 -137
19 6459
7106 2065
8447 -16
-5980 2259
-2008 -9692
-8112 2505
-2113 5674
7218 3305
-6240 -5917
1878 -6527
-5568 -7961
8259 -2469
4826 -6482
-2372 -8122
1228 2238
1492 6731
-5219 -1366
5665 -3253
-3818 175
-8344 -3991
2011 -2993
2953 -3602
853 -2326
9276 697
4214 -6075
-5201 -7484
-9451 -4934
-4009 8853
-4152 859
-2696 40
7198 -7840
-8905 4034
-6538 3143
-1111 -3427
5715 9004
-226 -7638
896 -9694
2851 -1428
-8542 4391
-2227 -6547
-8695 7004
8962 -2946
4699 -5017
1020 -4141
-9033 -1890
1070 91
-7301 -8234
-2042 -5139
-118 7663
6080 7921
3374 3741
9646 6386
-2034 1744
-4637 -7365
8953 -1932
-7014 -2878
-5209 -6145
-1724 3872
-1345 -3582
-5170 -2722
1132 6444
9786 -5960
-8966 -912
-9462 3471
8708 1246
1789 9336
-9395 1350
6457 -3256
-1369 7442
-8374 -1295
7180 -1785
8730 3842
7975 -3744
-3868 -7191
-3467 -3883
3479 -9679
4656 -8803
-8704 3903
4507 415
-8277 -8254
-8865 4814
322 7210
-7146 -9169
-4155 -8453
-4362 1941
