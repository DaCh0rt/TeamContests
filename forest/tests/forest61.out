81961.08 460164232.59
